thunder/core/symbol.py

```diff
@@ -21,7 +22,6 @@
 import thunder.core.dtypes as dtypes
 import thunder.core.devices as devices
 from thunder.core.proxies import Proxy, NumberProxy, variableify, CollectionProxy
-from thunder.core.utils import FrozenDict
```

Comment:
Why do you suggest that FrozenDict has no use here? How do you deal with keyword arguments names?

Reply:
I thought I could get away with flattening and keeping the tuple, however that's not the best idea as you pointed out in earlier comment.